### DISCUSSION TITLE
kubernetes minio ftp net port

### DISCUSSION THREAD

**Discussion by zzx2205710164** [minio/minio#17418] [Q&A] [answered]
Deploying minio to enable FTP service in k8s, -- ftp="address=: 8021" -- ftp="passive port range=30000-40000". How can ports 30000-40000 be mapped to external access ports of k8s when connecting to the mini FTP server using a client?
在k8s中部署minio开启ftp服务，--ftp="address=:8021" --ftp="passive-port-range=30000-40000"，当使用客户端连接minio ftp服务端时，怎么将端口30000-40000映射为k8s的外部访问端口呢？

**Comment by harshavardhana** [comment, marked as answer]
Just run on baremetal for ftp much simpler than working inside k8s.

Read about how you run run other normal ftp services inside k8s, if you can - you can do the same thing here.
